sycl/include/CL/sycl/detail/property_helper.hpp

```diff
@@ -40,6 +40,7 @@ enum PropWithDataKind {
   BufferContextBound,
   ImageUseMutex,
   ImageContextBound,
+  BufferMemChannel,
   PropWithDataKindSize
```

Comment:
Even though it's an internal enum, changing order of elements may break ABI. Suggest changing to:
```suggestion
  PropWithDataKindSize,
  BufferMemChannel
```

Reply:
@romanovvlad please take a look